backends/sycl-ref/ceed-sycl-ref-qfunction-load.hpp

```diff
@@ -8,6 +8,6 @@
 #ifndef _ceed_sycl_qfunction_load_hpp
 #define _ceed_sycl_qfunction_load_hpp
 
-CEED_INTERN int CeedQFunctionBuildKernel_Sycl(CeedQFunction qf);
+CEED_INTERN int CeedSyclBuildQFunction(CeedQFunction qf);
```

Comment:
```suggestion
CEED_INTERN int CeedQFunctionBuildKernel_Sycl(CeedQFunction qf);
```

We had a PR (https://github.com/CEED/libCEED/pull/1231) to make the function names here more consistent across backends. So I'm fairly certain we want to keep the original names.

Reply:
Yes, I retained the function names in main for the most part. Not sure how I missed this.

Reply:
Yeah, it looks like this PR missed updates in main. Thanks James for checking - this sort of consistency checking is tedious manual work but really helps maintainers when they need to go make a change across multiple backends